### PR TITLE
chore: Reworked the readme file

### PR DESCRIPTION
**Pull request by jo-gross**
## Closing issues:

- Closes: #84 

## Before you mark this PR as ready, please check the following points

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before
merging your code._

- [x] I´e created all necessary migrations?
- [x] The format is correct (`yarn format:check`, if invalid `yarn format:fix`)
- [x] No build errors (`yarn build`)
- [x] I´ve tested the implemented function by my own
- [x] Ensure the pr title fits the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) standard

## Describe your work, what changed

Changed the README to a project introduction instead of some personal comments

## Affected sites (please check during review):

- [ ] README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,231 @@
...
+
+> ![docker-compose](https://img.shields.io/badge/Docker-2496ED.svg?style=flat&logo=Docker&logoColor=white)
+>
+> ```sh
+> docker-compose up -d
+> ```
+
+#### Using `locally build`
+
+> ![docker-compose](https://img.shields.io/badge/Docker-2496ED.svg?style=flat&logo=Docker&logoColor=white)
```
Comment (jhns-de) [inline README.md:107]: two times the logo looks strange

File: README.md [hunk 2]
```diff
@@ -1,55 +1,231 @@
...
+
+---
+
+## 🧩 Features
+
+- **Multiple seperated Bars** - You can manage multiple bars with different recipes and users without sharing data
+  between them.
+- **User management** - Invite and manage users with different roles
+- **Cocktail recipe management** - Add, edit and delete cocktail recipes - in very detailed form.
+- **Optimal and optimized for bar tenders** - Option for easy search and compact view with the necessary steps for an
```
Comment (jhns-de) [inline README.md:52]: Bartender ist written without the space

File: README.md [hunk 3]
```diff
@@ -1,55 +1,231 @@
...
-```
+[3]: https://github.com/jo-gross/cocktail-manager/issues '🐛 Open an Issue'
+
+<p align="left">
+  <a href="https://github.com/jo-gross/cocktail-manager/graphs/contributors">
+    <img src="https://contrib.rocks/image?repo=jo-gross/cocktail-manager" />
+  </a>
+</p>
+
+As some asked, you´re also welcome to support financial to reduce costs, like hosting and domain fees, by donating.
```
Comment (jhns-de) [inline README.md:199]: Is there already some hosting and domains?

Comment (jo-gross) [reply README.md:199]: It´s in progress but not released yet. I can remove that part until its public available if you mind.

Comment (jo-gross) [reply README.md:199]: Currently hanging on legal stuff like privacy terms and terms of services (better save than sorry if i loose all of your fine developed recipes ;D)

Comment (jhns-de) [reply README.md:219]: I would remove it until there is something or just add the Github Sponsor button?

Comment (jo-gross) [reply README.md]: Alright

**Review by jhns-de — APPROVED** [review]

**Pull request merged**